api/ProgressEvent.json

```diff
@@ -92,6 +92,51 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "loaded_total_options_accept_double": {
+          "__compat": {
+            "description": "`loaded` and `total` options accept double",
```

Comment:
I'd maybe say "doubles" here, but otherwise, I think we're good @caugner, wdyt?

```suggestion
            "description": "`loaded` and `total` options accept doubles",
```